### PR TITLE
feat(databricks) : adding databricks pack

### PR DESCRIPTION
**Pull request by hsinghkalsi**
## Summary

Data-bricks pack to pull metrics from the data-bricks clusters using https://github.com/newrelic-experimental/nri-spark#databricks-init-script-creator-notebook 

### Screenshots

databricks

### CODE REVIEW

File: packs/databricks/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - databricks 
+
+# Authors of the pack (required)
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: Databricks init script creator notebook
+    description: >-
+      Databricks notebook to create int script to be used during intialization of databricks cluster
```
Comment (polfliet) [inline packs/databricks/config.yml:33]: small typos:
int -> init
intialization -> initialization
databricks -> DataBricks?

File: packs/databricks/dashboards/databricks.json [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+              "column": 2,
+              "row": 1,
+              "height": 2,
+              "width": 1
+            },
+            "title": "Spark Application Count",
+            "rawConfiguration": {
+              "nrqlQueries": [
+                {
+                  "accountId": 1606862,
```
Comment (polfliet) [inline packs/databricks/dashboards/databricks.json:39]: this needs to be set to 0, for all occurrences of accountId

**Review by polfliet — CHANGES_REQUESTED** [review]
Thanks for your contribution @hsinghkalsi !

You can always use the [importer script](https://github.com/newrelic/newrelic-observability-packs#testing) to test if the quickstart (dashboard creation) works.

Do you have any recommend alerts that you would contribute? Would be awesome to have both dashboards & alerts, if you have those available.

File: packs/databricks/config.yml (hunk 1 above)
Comment (hsinghkalsi) [reply packs/databricks/config.yml]: fixed, thanks for catching these.

File: packs/databricks/dashboards/databricks.json (hunk 2 above)
Comment (hsinghkalsi) [reply packs/databricks/dashboards/databricks.json]: fixed this

**Comment by polfliet** [comment]
Thanks for those updates @hsinghkalsi! I think the basis is looking good, some suggestions for improvement:
* I think we need a stronger description that speaks to the value statement and also helps with SEO, like what are the key metrics we have in that dashboard? @bradleycamacho do we have a template/example for that?
* Do you have a better screenshot? With some more interesting data rather than zeros.

**Comment by hsinghkalsi** [comment]
Thanks @polfliet , I have updated the snapshot
databricks

**Comment by polfliet** [comment]
Nice, that screenshot is looking great @hsinghkalsi !

**Review by polfliet — APPROVED** [review]

**Comment by bradleycamacho** [comment]
@hsinghkalsi @polfliet You can find a template and style tips in the [contributing file](https://github.com/newrelic/newrelic-observability-packs/blob/main/CONTRIBUTING.md#quickstarts-style-guide). Let me know if you need any assistance or guidance!

**Comment by jpvajda** [comment]
This  PR is almost ready to merge, it looks like the last request to 

> I think we need a stronger description that speaks to the value statement and also helps with SEO, like what are the key metrics we have in that dashboard? 

is still pending.

**Comment by hsinghkalsi** [comment]
@jpvajda  thanks for the nudge , I will have this done  today

**Comment by hsinghkalsi** [comment]
@jpvajda @polfliet Updated with stronger description.

**Comment by bradleycamacho** [comment]
@hsinghkalsi The description looks great! 

@jpvajda I'll do a formal review of the copy this afternoon and approve to merge.

**Review by bradleycamacho — APPROVED** [review]
I have made some copy edits to align this with the rest of our descriptions.

**Comment by jpvajda** [comment]
Thanks @hsinghkalsi and @bradleycamacho! This is all merged in now.

**Pull request merged**